lang/po/new/cs.po

```diff
@@ -93,7 +91,7 @@ msgstr "Dojely oba Z voziky k~hornimu dorazu?"
 # MSG_AUTO_HOME c=18
 #: messages.c:11
 msgid "Auto home"
-msgstr ""
+msgstr "Auto home"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.